view/frontend/templates/content/widget/sliders/list.phtml

```diff
@@ -0,0 +1,66 @@
+<?php /** @var \Blackbird\ContentManager\Block\Content\Widget\ContentList $block */ ?>
+<?php
+    $collection = $block->getCollection();
+    $link = $block->getLink();
+?>
+<div class="block widget contenttype-listing contents-grid grid">
+    <?php if (!empty($block->getTitle())): ?>
+        <div class="block-title">
+            <strong><?= $block->escapeHtml($block->getTitle()); ?></strong>
+        </div>
+    <?php endif; ?>
+    <div class="block-content">
+        <?php if ($collection->getSize()): ?>
+            <?php if ($block->hasPagerTop()): ?>
+                <div class="toolbar-bottom">
+                    <div class="toolbar top">
+                        <?= $block->escapeHtml($block->getPagerHtml()); ?>
+                    </div>
+                </div>
+            <?php endif; ?>
+
+            <div class="contents-items widget-content-grid">
+                <?php foreach ($collection as $content): ?>
+                    <div class="content-item">
+                        <?php // Has Link Top ?>
+                        <?php if ($block->hasLinkTop()): ?>
+                            <a class="contenttype-listing-readmore" href="<?= $block->escapeUrl($content->getLinkUrl()); ?>">
+                                <?= $block->escapeHtml(__($link['label'])); ?>
+                            </a>
+                        <?php endif; ?>
+
+                        <?php if (count($block->getAttributeToShow()) === 0): ?>
+                            <?= $block->escapeHtml($content->render('title', ['has_link' => true])); ?>
+                        <?php else: ?>
+                            <?php foreach ($block->getAttributeToShow() as $attribute): ?>
+                                <?= $block->escapeHtml($content->render($attribute['attribute'], $attribute['params'])); ?>
```

Comment:
not sure if we should escapeHtml here. Are there any attributes where html should be rendered?